cfme/tests/cloud_infra_common/test_snapshots_rest.py

```diff
@@ -188,7 +188,7 @@ def test_delete_snapshot_race(self, request, appliance, collection, vm):
                 raise
 
     @pytest.mark.rhv2
-    @pytest.mark.uncollectif(lambda provider: not provider.one_of(InfraProvider))
+    @pytest.mark.provider([VMwareProvider, RHEVMProvider], override=True)
```

Comment:
there is also SCVMM.  is it skipped somewhere ?

Reply:
Module marker does not include SCVMM, it specifically includes Openstack, Rhevm, VMware (and then the uncollect was uncollecting openstack). So Just RHEVM and VMWare here, though SCMM might be relevant. I'll leave that up to snapshots owner to add that coverage.